src/components/dialogs/parameters/parameters.js

```diff
@@ -378,7 +390,7 @@ export const useParametersBackend = (
     );
 
     useEffect(() => {
-        if (user !== null) {
+        if (user !== null && isAvailableService(type)) {
```

Comment:
There is something wrong here ... but i dont know what actually.
I said that because the providers retrieval does not work properly.

Reply:
fixed